app/assets/javascripts/search.js

```diff
@@ -87,6 +97,27 @@
         position = $link.closest('li').index() + startAt + 1; // +1 so it isn't zero offset
         GOVUK.analytics.callOnNextPage('setSearchPositionDimension', 'position=' + position + sublink);
       });
+    },
+    trackSearchResultsAndSuggestions: function ($searchResults) {
+      var searchResultData = {'urls': []},
+          searchURLs = search.extractSearchURLs($searchResults),
+          searchSuggestion = search.extractSearchSuggestion();
+
+      if (searchURLs.length) {
+        searchResultData['urls'] = searchURLs;
+      }
+
+      if (searchSuggestion !== null) {
+        searchResultData['suggestion'] = searchSuggestion;
+      }
+
+      if (GOVUK.analytics !== undefined &&
+          GOVUK.analytics.trackEvent !== undefined) {
+        GOVUK.analytics.trackEvent('Search Results', 'Shown', {
```

Comment:
From Tara's comment in the ticket:

    Category: searchResults
    Action: resultsShown

So I think this is should be `trackEvent('searchResults', 'resultsShown', {...`